src/style.css

```diff
@@ -1 +1,27 @@
-/* add styles here */
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
```

Comment:
To make it shorter you can use just - https://developer.mozilla.org/ru/docs/Web/CSS/background